dgraph/sample/data.json

```diff
@@ -14,175 +14,12 @@
           "value": "6.2.1 Beta-lactam medicines"
         },
         {
-          "description": "Oral dose forms",
```

Comment:
Think there might be some confusion between us on the schema? Have updated to what I think is the best/easiest at this stage. Feedback welcome :).

Reply:
I like this updated schema so far 👍 . (The two of us did discuss most of it in person but for posterity and to make sure I am on the same page 😄 )

Think these were the main schema/structure changes and rationale?
- form_category changed from a property to child relation 
  - Made going straight up/down the tree much easier (can recursively go down children to get to level of most detail rather than having to go via a property to get to a child) 
- Appending additional detail to description as we get further down the 'tree'
  - Dgraph description searching/building is now much easier (previously we had to build up the description string as we went down the tree on the client side, now we can just query for the specific node we want and use the description as is)
- Synonyms removed - problem for later? (Maybe to be added as properties?)

Reply:
> Synonyms removed

hm? They were in the wiki example, which is why I had them in. Similarly some of the other changes have now digressed from the wiki example. is there a reason for doing that?

Reply:
I think the removal of the Amox examples was an unintended side effect while restructuring - I'm happy to add those back in as a next step. I suppose it would also be good to get the rdf and json aligned as well?

Also think we should add the synonyms added back in - but not sure if we had a consensus on where they should go - do we still want them as properties of medicinal products?

Reply:
Personally, I think they should be properties. For the current schema, properties can't be nested, so if we want to had children off synonyms, they need to be entities (but IMO that will make things confusing).